policyengine_us/variables/gov/states/nd/tax/income/nd_taxable_income.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class nd_taxable_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "ND taxable income"
+    unit = currency-USD
```

Comment:
(Sorry, it differs from the parameter convention)
```suggestion
    unit = USD
```